doghouse/server/doghouse.go

```diff
@@ -0,0 +1,197 @@
+package server
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/google/go-github/github"
+	"github.com/haya14busa/reviewdog"
+	"github.com/haya14busa/reviewdog/diff"
+	"github.com/haya14busa/reviewdog/doghouse"
+)
+
+type Checker struct {
+	req *doghouse.CheckRequest
+	gh  *github.Client
+}
+
+func NewChecker(req *doghouse.CheckRequest, gh *github.Client) *Checker {
+	return &Checker{req: req, gh: gh}
+}
+
+func (ch *Checker) Check(ctx context.Context) (*doghouse.CheckResponse, error) {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method Checker.Check should have comment or be unexported